### DISCUSSION TITLE
Dokku don't crate logs for apps in /var/log/dokku/apps

### DISCUSSION THREAD

**Discussion by entyen** [dokku/dokku#6870] [Q&A]
Dokku don't crate logs for all apps in /var/log/dokku/apps i don't know why? where i can check or debug this plugin?

**Comment by josegonzalez** [comment]
Dokku doesn't write logs out to files by default. If you want to set this up, you'll need to setup the vector integration with a file sink for the logs. See the [vector log integration docs](https://dokku.com/docs/deployment/logs/#vector-logging-shipping) for more details.

**Reply by entyen** [reply]
I used this manual and setting up vector but don't get logs in dokku/apps folder

**Reply by josegonzalez** [reply]
Did you configure a file sink? Or just start vector?

**Reply by entyen** [reply]
![image](https://github.com/dokku/dokku/assets/31936997/72f6281e-a1b5-463b-9ff3-160d48d64cc5)
Configurated vector and start it,

**Reply by josegonzalez** [reply]
You don't have an app-specific log sink, and the global log sink is set to write to the console, or in this case, the vector container's standard out. You'll need to configure a log sink that writes to a file instead.

**Reply by entyen** [reply]
oh ok ty i find how)

dokku logs:set promtail vector-sink "file://?path=/var/log/dokku/apps/promtail.log&encoding[codec]=json"
